src/handlers/feature_handlers.cpp

```diff
@@ -4942,7 +4979,624 @@ void vehCarModelFeatureHandler::DrawShadow() {
 
 void vehCarModelFeatureHandler::DrawGlow() {
     auto model = reinterpret_cast<vehCarModel*>(this);
-    model->vehCarModel::DrawGlow();
+    if (!model->GetVisible())
+        return;
+
+    //get our geometry id
+    int geomSetId = model->GetGeomIndex();
+    int geomSetIdOffset = geomSetId - 1;
+
+    //get shaders
+    auto mainGeomEntry = lvlInstance::GetGeomTableEntry(geomSetIdOffset);
+    auto shaders = mainGeomEntry->pShaders[model->getVariant()];
+
+    //get car stuff we use to determine what to darw
+    auto car = model->getCar();
+    auto carsim = model->getCarSim();
+    auto siren = car->getSiren();
+    auto curDamage = car->getCarDamage()->getCurDamage();
+    auto maxDamage = car->getCarDamage()->getMaxDamage();
+    int gear = carsim->getTransmission()->getGear();
+    if (curDamage > maxDamage && vehCarModel::MWStyleTotaledCar)
+        return;
+
+    //setup renderer
+    Matrix44::Convert(gfxRenderState::sm_World, model->getCarMatrix());
+    gfxRenderState::m_Touched = gfxRenderState::m_Touched | 0x88;
```

Comment:
These two lines are another common operation which should be coverted into an inline function